### PR TITLE
Settings rearrange

### PR DESCRIPTION
**Pull request by rykelly**
Per GH-212. Should close it, I think.

### CODE REVIEW

File: db/R/get.trait.data.R [hunk 1]
```diff
@@ -142,6 +141,18 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon,
           for(id in 1:nrow(files)) {
             file.copy(file.path(files[[id, 'file_path']], files[[id, 'file_name']]), file.path(pft$outdir, files[[id, 'file_name']]))
           }
+          
+          # May need to symlink the generic post.distns.Rdata to a specific post.distns.*.Rdata file.
```
Comment (rykelly) [inline db/R/get.trait.data.R]: This block is unrelated (sorry) but makes meta-analysis update=AUTO work as expected.

File: db/tests/testthat/test.contents_sanity.R [hunk 2]
```diff
@@ -6,7 +6,7 @@
 # which accompanies this distribution, and is available at
 # http://opensource.ncsa.illinois.edu/license.html
 #-------------------------------------------------------------------------------
-con <- db.open(list(driver = "PostgreSQL", user = "bety", dbname = "bety", password = "bety"))
+con <- db.open(list(host="psql-pecan.bu.edu", driver = "PostgreSQL", user = "bety", dbname = "bety", password = "bety"))
```
Comment (rykelly) [inline db/tests/testthat/test.contents_sanity.R]: May want to reject this change (and next 3, which are equivalent) if it really makes more sense for tests to only pass if you have a local betyDB. But I needed to set the server for the tests to pass, and it seems like other users would too, typically.

File: settings/tests/testthat/testinput.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
   <bety>
     <user>bety</user>
     <password>bety</password>
-    <host>localhost</host>
+    <host>psql-pecan.bu.edu</host>
```
Comment (mdietze) [inline settings/tests/testthat/testinput.xml]: needs to be set back to localhost

**Comment by mdietze** [comment]
@tonygardella please update Gitbook to reflect changes in settings object.

**Comment by ankurdesai** [comment]
Also, I am modifying convert.input right now. Once merged, will need to resolve conflicts.

**Comment by rykelly** [comment]
Reverted to testing on localhost per @mdietze 's comment. Sorry I missed that...

**Pull request merged**